src/routes/HomePage/HomePage.js

```diff
@@ -42,6 +45,23 @@ export default function HomePage() {
     }
   }, [isMobile]);
 
+  // This useEffect only runs one when mounted
+  useEffect(() => {
```

Comment:
We already have `useEffect` line 24. Can we move to line 24? Let me know if I am missing something!

Reply:
Good call. For some reason I though it wouldn't work, but it did. Updated!